examples/node_disk_failure.yaml

```diff
@@ -0,0 +1,22 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2023 Datadog, Inc.
+
+apiVersion: chaos.datadoghq.com/v1beta1
+kind: Disruption
+metadata:
+  name: disk-failure
+  namespace: chaos-demo
+spec:
+  level: node # impact the whole node instead of a single pod
+  selector:
+    kubernetes.io/hostname: lima-default # when node level is enabled, the selector applies to nodes and not pods
+  #level: pod
```

Comment:
The commented out options should be removed

Reply:
Oups... I will clean this file 👍